svc/crawlers/elephant_scrapper/pipelines.py

```diff
@@ -0,0 +1,55 @@
+# Define your item pipelines here
+#
+# Don't forget to add your pipeline to the ITEM_PIPELINES setting
+# See: https://docs.scrapy.org/en/latest/topics/item-pipeline.html
+
+
+# useful for handling different item types with a single interface
+from itemadapter import ItemAdapter
+from scrapy.exceptions import DropItem
+
+
+class FilterImcompleteDataPipeline:
```

Comment:
scrapy does not have such interface.
I don't know to define class Pipeline, because scrapy uses some hooks, only when they are defined to class, and if I try to define Abstract class Pipeline, I will need to define all hooks, that breaks scrapy

Reply:
Then let's god be with that thing, at the end of the day it is a small project. Redesigning all things might be a little bit an overhead))) I guess we can live without `Pipeline`)))